doc/cephadm/install.rst

```diff
@@ -108,54 +119,67 @@ This command will:
 
 * Create a monitor and manager daemon for the new cluster on the local
   host.
-* Generate a new SSH key for the Ceph cluster and adds it to the root
+* Generate a new SSH key for the Ceph cluster and add it to the root
   user's ``/root/.ssh/authorized_keys`` file.
-* Write a minimal configuration file needed to communicate with the
-  new cluster to ``/etc/ceph/ceph.conf``.
+* Write a minimal configuration file to ``/etc/ceph/ceph.conf``. This
+  file is needed to communicate with the new cluster.
 * Write a copy of the ``client.admin`` administrative (privileged!)
   secret key to ``/etc/ceph/ceph.client.admin.keyring``.
-* Write a copy of the public key to
-  ``/etc/ceph/ceph.pub``.
+* Write a copy of the public key to ``/etc/ceph/ceph.pub``.
 
-The default bootstrap behavior will work for the vast majority of
-users.  See below for a few options that may be useful for some users,
-or run ``cephadm bootstrap -h`` to see all available options:
+Further information about cephadm bootstrap 
+-------------------------------------------
 
-* In larger Ceph clusters, network separation between the public
-  network traffic and cluster traffic which handles replication,
-  recovery and heartbeats between OSD daemons, can lead to performance
-  improvements. To define the `cluster network`_ you can supply the
-  ``--cluster-network`` option to the ``bootstrap`` subcommand. This
-  parameter must define a subnet in CIDR notation, for example
-  10.90.90.0/24 or fe80::/64.
+The default bootstrap behavior will work for most users. But if you'd like
+immediately to know more about ``cephadm bootstrap``, read the list below.  
 
-* Bootstrap writes the files needed to access the new cluster to ``/etc/ceph``,
-  so that any Ceph packages installed on the host itself (e.g., to access the
-  command line interface) can easily find them.
+Also, you can run ``cephadm bootstrap -h`` to see all of ``cephadm``'s
+available options.
+
+* Larger Ceph clusters perform better when (external to the Ceph cluster)
+  public network traffic is separated from (internal to the Ceph cluster)
+  cluster traffic. The internal cluster traffic handles replication,
+  recovery, and heartbeats between OSD daemons.  You can define the `cluster
+  network`_ by supplying the ``--cluster-network`` option to the
```

Comment:
https://ceph--39760.org.readthedocs.build/en/39760/cephadm/rados/configuration/network-config-ref#cluster-network

404 not found

Reply:
Good find.

Check this out: It's not on the live site either:

https://docs.ceph.com/en/latest/cephadm/rados/configuration/network-config-ref#cluster-network

Reply:
(Give me a few minutes to test this locally, and expect another push.)

Reply:
Okay. This is now patched up. I'm resolving the conversation and asking for another review.